### PR TITLE
docs: add simulate dns chaos on kubernetes

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
Signed-off-by: Liuxiaozhen12 <82579298+Liuxiaozhen12@users.noreply.github.com>
This PR is translated from: https://github.com/chaos-mesh/website-zh/pull/29

### CODE REVIEW

File: docs/simulate-dns-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Open Chaos Dashboard, and click **NEW EXPERIMENT** on the page to create a new experiment:
+
+   ![Create Experiment](./img/create-new-exp.jpeg)
+
+2. In the **Choose a Target** area, choose **DNS FAULT** and select a specific behavior, such as **ERROR**. Then fill out the matching rules.
+
+   ![DNSChaos Experiment](./img/dnschaos-exp.png)
+
+   According to the matching rules configured in the screenshot, the DNS FAULT takes effect for domains including `google.com`, `chaos-mesh.org`, and `github.com`, which means that an error will be returned when a DNS request is sent to these three domains.For details of specific matching rules, refer to the description of the `patterns` field in [Configuration Description](#configuration description).
```
Comment (WangXiangUSTC) [inline docs/simulate-dns-chaos-on-kubernetes.md:48]:
```suggestion
   According to the matching rules configured in the screenshot, the DNS FAULT takes effect for domains including `google.com`, `chaos-mesh.org`, and `github.com`, which means that an error will be returned when a DNS request is sent to these three domains. For details of specific matching rules, refer to the description of the `patterns` field in [Configuration Description](#configuration description).
```

File: docs/simulate-dns-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+|:--------- |:------------ |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |:------------- |:-------- |:-------------------------------------------- |
+| action    | string       | Defines the behavior of DNS fault. The value can be `random` or `error`.When the value is `random`, DNS service will return a random IP address; when the value is `error`, DNS service will return an error.                                                                                                                                                               | None          | Yes      | `random` or `error`                          |
```
Comment (WangXiangUSTC) [inline docs/simulate-dns-chaos-on-kubernetes.md:90]:
```suggestion
| action    | string       | Defines the behavior of DNS fault. The value can be `random` or `error`. When the value is `random`, DNS service will return a random IP address; when the value is `error`, DNS service will return an error.                                                                                                                                                               | None          | Yes      | `random` or `error`                          |
```

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM, and need to fix ci

**Comment by g1eny0ung** [comment]
@Liuxiaozhen12 This PR should request a merge to the `2.0` branch, not `master`. Need to resubmit a PR.

**Pull request closed without merging**